lib/xcodeproj/constants.rb

```diff
@@ -104,6 +104,7 @@ module Constants
       :dynamic_library   => 'com.apple.product-type.library.dynamic',
       :static_library    => 'com.apple.product-type.library.static',
       :bundle            => 'com.apple.product-type.bundle',
+      :octest_bundle     => 'com.apple.product-type.bundle',
```

Comment:
Note that this wasn't necessary before because [we did *hardcode* that extension in the code](https://github.com/CocoaPods/Xcodeproj/pull/288/files#diff-b65790afba1783cb56920e9038652be6L111).
Which also mean that with the previous code and `XCScheme::add_test_target` method, it seems that we were only able to create OCUnit test targets (and not XCUnit test targets), right?